project/Versions.scala

```diff
@@ -17,7 +17,7 @@ import scala.util.Properties
  */
 object Versions {
 
-  val crossScala = Seq("2.11.5", "2.10.4")
+  val crossScala = Seq("2.11.6", "2.10.5")
```

Comment:
We would not. After writing that I realized it would cause more travis builds. But I never deleted my above comment.